qiskit_nature/problems/sampling/protein_folding/exceptions/invalid_side_chain_exception.py

```diff
@@ -0,0 +1,18 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An exception for an invalid side chain in a peptide."""
+
+
+class InvalidSideChainException(Exception):
```

Comment:
Just wondering how useful it is to have these various subclasses. Ought they to be subclasses of QiskitNatureError if they are needed.

Reply:
It is to make it more explicit what type of error occurs and to potentially catch only certain types of errors.